corehq/apps/users/templates/users/web_users.b3.html

```diff
@@ -121,65 +131,145 @@
             using the CommCare Application.
         {% endblocktrans %}
         <div class="btn-toolbar">
-            <a class="btn btn-success" href="{% url "invite_web_user" domain %}">
-                <i class="fa fa-plus"></i> {% trans 'Invite Web User' %}</a>
+            <a class="btn btn-success"
+               href="{% url "invite_web_user" domain %}">
+                <i class="fa fa-plus"></i> {% trans 'Invite Web User' %}
+            </a>
         </div>
     </p>
+    <div ng-app="usersApp">
+        <div class="panel panel-default" ng-controller="paginatedListController">
+            <div class="panel-heading">
+                <div class="row">
+                    <div class="col-sm-6">
+                        <h3 class="panel-title" style="padding-top: 7px;">{% trans 'Current Users' %}</h3>
+                    </div>
+                    <div class="col-sm-6">
+                        <div class="input-group">
+                            <input type="text"
+                                   class="form-control"
+                                   ng-model="query"
+                                   placeholder="{% trans "Filter Users..." %}">
+                            <span class="input-group-btn">
+                                <button class="btn btn-default"
+                                        ng-click="filterUsers()"
+                                        type="button">
+                                    <i class="fa fa-search"></i>
+                                </button>
+                            </span>
+                        </div><!-- /input-group -->
+                    </div>
+                </div>
+            </div>
+            <div class="panel-body">
+                <table class="table table-striped table-responsive"
+                       style="margin-bottom: 0;">
+                    <thead>
+                        <tr>
+                            <th>{% trans "E-mail" %}</th>
+                            <th>{% trans "Role" %}</th>
+                            <th>{% trans "Name" %}</th>
+                            <th>{% trans "Phone Numbers" %}</th>
+                            <th>{% trans "Remove" %}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr ng-show="notLoaded">
+                            <td colspan="5">{% trans "Loading Users..." %}</td>
+                        </tr>
+                        <tr ng-repeat="user in paginatedItems">
+                            {% angularjs %}
```

Comment:
How's this work again?  all the ``{{ vars }}`` inside this tag aren't rendered by the django template, so angular takes over?